TicTacToe/TicTacToe.py

```diff
@@ -0,0 +1,152 @@
+def get_board_size():
+    """ Board must be squared and and odd (with one center """
+    while True:
+        try:
+            print("Enter board Size (odd number)")
+            board_size = int(input())
+        except ValueError:
+            print("Board size must be a Number")
+            continue
+        if(board_size % 2 == 0 ):
+            print("Number must be odd")
+        # is odd number
+        else:
+            break
+    return board_size
+
+
+def init_board(board_size):
+    """ print blanked board with '0' in each cell """
+    board = [[0 for cell in range(0,board_size)] for row in range(0,board_size)]
+    return board
+
+
+def print_board_state(board):
+    """ Draw the board after each turn """
+    for row in board:
+        print(" -"*len(board))
+        print("|", end='')
+        for cell  in row:
+            if cell  == 0:
+                print(" ", end='')
+            else:
+                print(cell , end='')
+            print("|",end='')
+        print("")
+        print(" -"*len(board))
+
+
+def choose_player(board):
```

Comment:
wow this looks like a lot of work to choose player. 
there is a much much simple way to do it.
think of it.